.travis.yml

```diff
@@ -16,7 +16,7 @@ cache:
     - $HOME/.composer/cache
 
 php:
-  - 7.4
```

Comment:
Why drop `7.4`? It's needed for the code coverage builds.

Reply:
You probably want to rather add 3 matrix includes below.

Reply:
This is just a draft to test.